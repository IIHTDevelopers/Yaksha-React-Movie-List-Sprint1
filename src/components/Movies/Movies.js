import React from 'react'
import styles from './Movies.module.css'
import { db } from '../../data-source/movie-db'
import Card from '../Card/Card'
import Checkbox from '../Checkbox/Checkbox'

const Movies = () => {
  

  return (
    <div>
      <p>Movies</p>
    </div>
  )
}

export default Movies
