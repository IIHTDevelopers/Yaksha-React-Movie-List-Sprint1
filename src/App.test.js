import React from 'react'
import { shallow } from 'enzyme/build'
import App from './App';
import Movies from './components/Movies/Movies';

let testName = "AppTest boundary"

describe("boundary", () => {
  it(`${testName} should mount App without crashing`, () => {
    const wrapper = shallow(<App />)
    expect(wrapper.getElements()).toMatchSnapshot();
    wrapper.unmount()
  })

  it(`${testName} should mount Movie without crashing`, () => {
    const wrapper = shallow(<Movies />)
    expect(wrapper.getElements()).toMatchSnapshot();
    wrapper.unmount()
  })

})