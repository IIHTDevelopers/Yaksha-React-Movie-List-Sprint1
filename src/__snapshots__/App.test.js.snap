// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`boundary AppTest boundary should mount App without crashing 1`] = `
Array [
  <div
    className="App"
  >
    <Movies />
  </div>,
]
`;

exports[`boundary AppTest boundary should mount Movie without crashing 1`] = `
Array [
  <div
    className="Movies"
  >
    <h3>
      Movies
    </h3>
    <fieldset>
      <legend>
        Filters
      </legend>
      <div>
        <label>
          <b>
            Genres
          </b>
        </label>
      </div>
      <div>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "action",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "fantasy",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "romance",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "horror",
              }
            }
          />
        </React.Fragment>
      </div>
      <div>
        <label>
          <b>
            Language
          </b>
        </label>
      </div>
      <div>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "hindi",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "english",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "telugu",
              }
            }
          />
        </React.Fragment>
        <React.Fragment>
          <Checkbox
            item={
              Object {
                "checked": true,
                "name": "punjabi",
              }
            }
          />
        </React.Fragment>
      </div>
    </fieldset>
    <Card
      movie={
        Object {
          "available": true,
          "genre": "fantasy",
          "id": 1,
          "image": "https://i.picsum.photos/id/650/300/200.jpg?hmac=nuRnzrByBYKU6fsTl1xwz3MIpNrNjb0PjLJA5ypiOqU",
          "language": "Telugu",
          "length": 119,
          "price": 300,
          "ratings": 3.4,
          "releasedDate": "28/12/2012",
          "showTime": "21:00",
          "title": "Superman is flying",
        }
      }
    />
    <Card
      movie={
        Object {
          "available": true,
          "genre": "horror",
          "id": 2,
          "image": "https://i.picsum.photos/id/650/300/200.jpg?hmac=nuRnzrByBYKU6fsTl1xwz3MIpNrNjb0PjLJA5ypiOqU",
          "language": "English",
          "length": 124,
          "price": 370,
          "ratings": 3.9,
          "releasedDate": "14/7/2014",
          "showTime": "15:00",
          "title": "Batman is watching",
        }
      }
    />
  </div>,
]
`;
